### PR TITLE
Discount Management edits

### PR DESCRIPTION
**Pull request by AlexSlawinski**
review of discount management

## PR Description

TBD

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/pbc/all/discount-management/import-and-export-data/file-details-discount-store.csv.md [hunk 1]
```diff
@@ -29,8 +29,8 @@ data:import:discount-store
 
 | PARAMETER | REQUIRED | TYPE | REQUIREMENTS OR COMMENTS | DESCRIPTION |
 | --- | --- | --- | --- | --- |
-| discount_key | &check; | String |`discount_key` must be included in the `discount.csv` file. |  |
-| store_name | &check; | String |  | Name of the store to which the discount applies. |
+| discount_key | &check; | String |A `discount_key` must be included in the `discount.csv` file. |  |
```
Comment (VadymSachenko) [inline docs/pbc/all/discount-management/import-and-export-data/file-details-discount-store.csv.md:32]: `discount_key` is a parameter name, so we shouldn't use the article in this case. So, it should be whether `discount_key` or the `discount_key` parameter.
```suggestion
| discount_key | &check; | String |`discount_key` must be included in the `discount.csv` file. |  |
```

File: docs/pbc/all/discount-management/import-and-export-data/file-details-discount-voucher.csv.md [hunk 2]
```diff
@@ -29,13 +29,13 @@ data:import:discount-voucher
...
-| random_generated_code_length | &check; | String |If quantity >= 1 then `random_generated_code_length`	cannot be empty. | Random part of the voucher code with the amount of random symbols equals to the value of `random_generated_code_length` field. |
-| max_number_of_uses |  | Number |If empty it will be set to 0. | Maximum number of this voucher usage. |
-| voucher_batch | &check; | Number |`voucher_batch` must be previously created during *discount.csv* import, then the batch value must be a different number for each row in the file. | Voucher batch groups vouchers into batches. It identifies a voucher belonging to the same voucher pool. |
-| is_active |  | Boolean | If empty, will be set to False = 0.<ul><li>True = 1</li><li>False = 0</li></ul>  | Indicates if discount voucher is active or not. |
+| discount_key | &check; | String |A `discount_key` must exist in the `discounts.csv` file | The key identifier of the discount. |
```
Comment (VadymSachenko) [inline docs/pbc/all/discount-management/import-and-export-data/file-details-discount-voucher.csv.md:32]: same
```suggestion
| discount_key | &check; | String |`discount_key` must exist in the `discounts.csv` file | The key identifier of the discount. |
```

File: docs/pbc/all/discount-management/manage-via-glue-api/manage-discounts-via-glue-api.md [hunk 3]
```diff
@@ -18,15 +18,15 @@ related:
     link: docs/scos/user/features/page.version/promotions-discounts-feature-overview.html
 ---
 
-The [Promotions](/docs/pbc/all/discount-management/discount-management.html) functionality enables sellers to provide a promotional item that the customers can add to their carts at a discounted price or even for free. To be eligible for promotions, the purchase needs to fulfill certain discount conditions, for example, the purchase amount should exceed a certain threshold.
+The [Promotions](/docs/pbc/all/discount-management/discount-management.html) functionality enables sellers to provide a promotional item that the customers can add to their carts at a discounted price, or even for free. To be eligible for promotions, the purchase needs to fulfill certain discount conditions, for example the purchase amount exceeding a certain threshold.
```
Comment (VadymSachenko) [inline docs/pbc/all/discount-management/manage-via-glue-api/manage-discounts-via-glue-api.md:21]: enable => let - see https://developers.google.com/style/word-list#enable
```suggestion
The [Promotions](/docs/pbc/all/discount-management/discount-management.html) functionality lets sellers provide a promotional item that the customers can add to their carts at a discounted price or even for free. To be eligible for promotions, the purchase must fulfill certain discount conditions—for example, the purchase amount exceeding a certain threshold.
```

**Review by VadymSachenko — COMMENTED** [review]
@AlexSlawinski please check links I've shared with you and go through the documents again and make sure they follow Google style guide:
Make the following adjustments:
Details/information on => Details/information on
Enable someone to => let someone
Future tense => Present tense (where applicable)
below => following

When you're done, let me know and I'll check the PR again. Thank you for you effort and time!

**Review by VadymSachenko — CHANGES_REQUESTED** [review]
@AlexSlawinski please check links I've shared with you and go through the documents again and make sure they follow Google style guide:
Make the following adjustments:
Details/information on => Details/information on
Enable someone to => let someone
Future tense => Present tense (where applicable)
below => following

When you're done, let me know and I'll check the PR again. Thank you for you effort and time!

File: docs/pbc/all/discount-management/manage-via-glue-api/manage-discounts-via-glue-api.md [hunk 4]
```diff
@@ -18,15 +18,15 @@ related:
...
 
 {% info_block infoBox "Info" %}
 
-For more details on how to create the discount conditions, see [Create discounts](/docs/pbc/all/discount-management/manage-in-the-back-office/create-discounts.html).
+For more details about how to create the discount conditions, see [Create discounts](/docs/pbc/all/discount-management/manage-in-the-back-office/create-discounts.html).
 
 {% endinfo_block %}
 
-In your development, the Promotions API will help you to enable customers to redeem the benefits provided by promotions and check the correct order amount, discount included.
+In your development, the Promotions API helps you enable customers to redeem the benefits provided by promotions and check the correct order amount with the discount included.
```
Comment (AlexSlawinski) [inline docs/pbc/all/discount-management/manage-via-glue-api/manage-discounts-via-glue-api.md:29]:
```suggestion
In your development, the Promotions API helps you to allow customers to redeem the benefits provided by promotions and check the correct order amount with the discount included.
```

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**